person.rb

```diff
@@ -0,0 +1,25 @@
+# person.rb
+
+class Person
+  attr_reader :id, :name, :age
```

Comment:
- [ ] Good job so far 👍🏽. I think it will be better if the name and the age parameters are accessible outside the class so that they can easily be modified. therefore, I will recommend you use an attribute accessor for the name and age.
e.g.
`attr_accessor :name, :age `